mmdet/datasets/pipelines/loading.py

```diff
@@ -3,6 +3,7 @@
 import mmcv
 import numpy as np
 import pycocotools.mask as maskUtils
+from panopticapi.utils import rgb2id
```

Comment:
Same as above

Reply:
We can use `try` here to allow users to use our codes normally without installing `panopticapi`.